__tests__/app.test.js

```diff
@@ -24,8 +24,46 @@ describe("nc-games app", () => {
         });
     });
   });
+  describe("GET /api/reviews/:review_id", () => {
+    it("200 OK: returns a review object", () => {
+      return request(app)
+        .get("/api/reviews/5")
+        .expect(200)
+        .then(({ body: { review } }) => {
+          expect(review).toEqual(
+            expect.objectContaining({
+              review_id: expect.any(Number),
```

Comment:
This is great. Just one comment: review_id should not be any number, it has to match the review_id we are trying to request (5 in this case!) :)